doc/source/cluster/running-applications/job-submission/openapi.yml

```diff
@@ -0,0 +1,471 @@
+                $ref: '#/components/schemas/JobStatus'
+        500:
+          summary: Internal Error
+          description: |
+            An internal error occurred when stopping the job.
+          content:
+            text/plain:
+              schema:
+                description: The error message.
+                type: string
+        404:
+          summary: Job Not Found
+          description: |
+            The job does not exist.
+          content:
+            text/plain:
+              schema:
+                description: The error message.
+                type: string
+        400:
+          summary: Wrong Job Type
+          description: |
+            The job was not submitted via the Ray Jobs API, so it cannot be stopped.
+          content:
+            text/plain:
+              schema:
+                description: The error message.
+                type: string
+  /api/jobs/{submission_id}/logs:
+    get:
+      summary: Get Job Logs
+      description: |
+        Get the logs of a job.
+      parameters:
+        - name: submission_id
+          in: path
+          description: |
+            The ID of the job to get the logs of.
+          required: true
+          schema:
+            type: string
+      responses:
+        200:
+          summary: Job Logs
+          description: |
+            The logs of the job.
+          content:
+            application/json:
+              schema:
+                type: object
+                properties:
+                  logs:
+                    type: string
+                    description: The logs of the job.
+        500:
+          summary: Job Logs Internal Error
+          description: |
+            An internal error occurred when getting the logs of the job.
+          content:
+            text/plain:
+              schema:
+                description: The error message.
+                type: string
+        404:
+          summary: Job Not Found
+          description: |
+            The job does not exist.
+          content:
+            text/plain:
+              schema:
+                description: The error message.
+                type: string
+        400:
+          summary: Wrong Job Type
+          description: |
+            The job was not submitted via the Ray Jobs API, so its logs cannot be retrieved.
+          content:
+            text/plain:
+              schema:
+                description: The error message.
+                type: string
+  /api/jobs/{submission_id}/logs/tail:
+    get:
+      summary: Tail Job Logs
+      description: |
+        WebSocket endpoint for tailing the logs of a job 
+        (Not documented in OpenAPI, see 
+        https://docs.ray.io/en/latest/_modules/ray/dashboard/modules/job/sdk.html#JobSubmissionClient.tail_job_logs
+        for example usage).
+      parameters:
+        - name: submission_id
+          in: path
+          description: |
+            The ID of the job to tail the logs of.
```

Comment:
Ending with a proposition could read awkward in some cases. Would this be simpler: "Tail job ID's logs."